src/EnergyPlus/DataGlobals.hh

```diff
@@ -174,6 +174,7 @@ struct DataGlobal : BaseGlobalStruct
     bool printConsoleOutput = true;
     bool installRootOverride = false;
     int numThread = 1;
+    bool AirLoopHVACDOASUsedInSim = false;
```

Comment:
We can always keep discussing the best place for these flags, but this is a totally reasonable place to drop it for now.  This flag is intended to be a 'global' flag for a given running simulation.  So DataGlobals is good.